app/styles/_utils.less

```diff
@@ -16,3 +16,8 @@
 .hide-if-empty:empty {
   display: none !important;
 }
+
+// generic classes not included with twbs
+.inline-block {
+  display:inline-block;
+}
```

Comment:
I agree in that I think it's be better to create a consistent markup pattern with classes semantically describing the elements (separating content from presentation) rather than putting helper classes on elements as needed (putting presentational classes in the content).